pkg/cronjob/spark_dependencies.go

```diff
@@ -50,6 +52,12 @@ func CreateSparkDependencies(jaeger *v1.Jaeger) *batchv1beta1.CronJob {
 
 	commonSpec := util.Merge([]v1.JaegerCommonSpec{jaeger.Spec.Storage.Dependencies.JaegerCommonSpec, jaeger.Spec.JaegerCommonSpec, baseCommonSpec})
 
+	image := jaeger.Spec.Storage.Dependencies.Image
+	if image == "" {
+		// the version is not included, there is only one version - latest
+		image = viper.GetString("jaeger-spark-dependencies-image")
```

Comment:
> codecov doesn't seem related to this PR ?

This does not seem to be covered.

Reply:
This is tested in [both of these tests](https://github.com/jaegertracing/jaeger-operator/blob/e117f178679213b5bd595f37df43c86961dffbe7/pkg/cronjob/spark_dependencies_test.go#L207-L226).